refactor: :recycle: add book parts "Workshop overview" and "Pre-workshop tasks" (#122)
rostools/github-intro: pre-workshop/index.qmd

in this section and finish with **completing the
[survey](/pre-workshop/pre-survey.qmd)** at the end. These tasks are
designed to make it easier for everyone to start the workshop with
everything set up and a shared foundation to build on.

Depending on your skills and knowledge, these tasks could take between
**1-2 hrs to finish**, so we suggest planning some time to complete
them. While many of the tasks are mostly readings, they are important in
making sure you get the most out of the workshop.

## Summary of tasks

Here's a quick overview of the tasks you need to do. Specific details
about them are found as you work through the tasks.

1.  If you haven't already, go through the workshop overview section,
    including the [Syllabus](/overview/syllabus.qmd), [Is this for
    you?](/overview/is-it-for-you.qmd), and [Learning
    design](/overview/learning-design.qmd) to get an overview of what
    this workshop contains and the learning design behind the workshop
    material.
2.  Go through the the pre-workshop tasks to help build a shared
    foundation to build on when the workshop starts. This includes
    reading the [Code of conduct](/pre-workshop/code-of-conduct.qmd) and

>     [Introduction to version control, Git and
>     GitHub](/pre-workshop/git-and-github.qmd), as well as creating a

Reviewer: ```suggestion
    [Version control with Git and
    GitHub](/pre-workshop/git-and-github.qmd), as well as creating a
```

    [GitHub user account](/pre-workshop/github-account.qmd). Remember to

>     finish the pre-workshop [survey](/pre-workshop/pre-survey.qmd) at

Reviewer: I wonder if this might be better as a simple list like:

```
Read: 

1. blah
2. blah
```

Or even a table?
